docs/concepts/hosts.mdx

```diff
@@ -0,0 +1,53 @@
+---
+title: "Hosts"
+icon: "ti-anchor" # themify icon pack : https://themify.me/themify-icons
+description: "The runtime environment for actors and capability providers"
+sidebar_position: 2
+type: "docs"
+---
+
+## Overview
+
+The wasmCloud **host** is an application runtime. The host is responsible for loading, instantiating, and communicating among WebAssembly components and capability providers, in a secure, distributed way.
+
+### Cattle, Not Pets
+
+A core design principle of wasmCloud is that the health of the lattice and applications running on it should not be dependent on any particular host. Hosts are designed to be interchangeable and should be able to start and stop without causing downtime. While it's possible to run a single host in a lattice, it's also possible for workloads to be distributed across many hosts, and for those hosts to be dynamically scaling up and down in response to changing demand.
+
+### A Secure Sandbox
+
+WebAssembly components require a runtime to execute. The wasmCloud host is **not** a WebAssembly runtime. The wasmCloud host uses [Wasmtime](https://github.com/bytecodealliance/wasmtime), a fast, secure, and standards-compliant runtime that supports the [Component Model](https://github.com/WebAssembly/component-model), as its WebAssembly runtime. wasmCloud adds additional layers of [security](/docs/hosts/security) on top of the runtime as part of its zero trust security model.
+
+### RPC as a Bridge to Capabilities
+
+wasmCloud [actors](./actors.mdx) cannot directly access capabilities. Instead, the host subscribes to incoming messages and makes outbound requests on behalf of actors, using the [wasmCloud RPC protocol](/docs/hosts/lattice-protocols/rpc). This removes the need for actors (and developers!) to have any knowledge of the underlying transport mechanism, and allows the host to run untrusted components without compromising security.
+
+### The Control Interface
+
+The API for managing hosts, their inventories (actors and providers), and host/lattice metadata is a [set of NATS subscriptions](/docs/hosts/lattice-protocols/control-interface). A separate server is not needed to manage hosts and the lattice. Instead, each host within a lattice subscribes to NATS subjects corresponding to its own responsibilities, as well as lattice-wide subjects.
+
+### Events
+
+Hosts will emit [CloudEvents](https://cloudevents.io/) during normal operation, as well as in response to errors. These [events](/docs/reference/cloud-event-list) reflect coarse-grained state changes for a lattice. This event log is useful for monitoring and observability, and is used internally by [wadm](/docs/category/declarative-application-deployment-wadm) to track whether applications have been successfully deployed.
+
+### Distributed Observability
```

Comment:
This should be higher up IMO -- o11y is a polish point that makes sense to show off